modules/ROOT/pages/as2/as2-connectorMule4-60.adoc

```diff
@@ -0,0 +1,80 @@
+= AS2 Connector - Get Started - Mule 4
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Support Category: https://www.mulesoft.com/legal/versioning-back-support-policy#anypoint-connectors[Premium]
+
+AS2 Connector Version 6.0.0
+
+Mule runtime engine (Mule) supports the AS2 protocol through the Anypoint Connector for AS2 (AS2 Connector).
+
+The AS2 Connector enables trading partners to transmit and receive AS2 messages with EDI-X12, EDIFACT, XML, or binary payloads. For example:
+
+* Send messages, signed or unsigned, over HTTP or HTTPS, following the AS2 protocol
+
+* Receive messages, also over HTTP or HTTPS, following the AS2 protocol
+
+* Receive asynchronous MDN for the message sent
+
+* Storing signed copies of receipts and original message exchanges for non-repudiation purposes
+
+Release Notes: xref:release-notes::connector/as2-connector-release-notes-mule-4.adoc[AS2 Connector Release Notes]
+
+== About Connectors
+
+Anypoint connectors are Mule runtime engine extensions that enable you to connect
+to APIs and resources on external systems, such as X12 and RosettaNet.
+
+== Prerequisites
+
+Before creating an app, you must have access to the AS2 target resource and
+Anypoint Platform. You also need understand how to create a Mule app using Anypoint Studio.
+You should have familiarity with AS2.
+
+=== Considerations when Migrating from the AS2 connector 5.x
+
+The AS2 version 6 connector requires Mule 4.3.0 as minimum runtime version. Refer to xref:as2/as2-connector-upgrade-migrate-6-0.adoc[Upgrade and Migrate] for more information.
+
+== Audience
+
+* Starting user -- To create your Mule app,
+see xref:as2/AS2-connector-studio-6-0.adoc[Studio Configuration]. The
+xref:as2/AS2-connector-examples-6-0.adoc[Examples] topic provides one or more use cases for using the connector.
+* Power user - Read the xref:as2/AS2-connector-xml-maven-6-0.adoc[XML and Maven Support],
+and xref:as2/AS2-connector-examples-6-0.adoc[Examples] topics.
+
+== Common Use Cases For the Connector
```

Comment:
```suggestion
== Common Use Cases for the Connector
```

Reply:
The following do not look like use cases; they look like uses for or capabilities.
A use case typically has more detail or specifics, like a scenario.